process/src/main/java/org/openjdk/skara/process/Process.java

```diff
@@ -67,6 +67,13 @@ public Description environ(String key, String value) {
             return this;
         }
 
+        public Description environMap(Map<String, String> keyValueMap) {
```

Comment:
I would have just overloaded `environ`, as in `public Description environ(Map<String, String> env)`